pydoctor/astbuilder.py

```diff
@@ -919,12 +959,13 @@ def _annotation_from_attrib(self,
 
     def _annotations_from_function(
             self, func: Union[ast.AsyncFunctionDef, ast.FunctionDef]
-            ) -> Mapping[str, Optional[ast.expr]]:
+            ) -> Tuple[Mapping[str, Optional[ast.expr]], Optional[ast.expr]]:
         """Get annotations from a function definition.
         @param func: The function definition's AST.
-        @return: Mapping from argument name to annotation.
-            The name C{return} is used for the return type.
```

Comment:
We can probably keep the older way of storing the return annotation, since parameters can’t be called « return », it’s probably better not to change this needlessly.

this will also make the some code more simple.